flexget/plugins/sites/torrentday.py

```diff
@@ -155,7 +164,7 @@ def search(self, task, entry, config=None):
                 entry['url'] = torrent_url
 
                 # us tr object for seeders/leechers
-                seeders, leechers = tr.find_all('td', { 'class': ['seedersInfo', 'leechersInfo']})
+                seeders, leechers = tr.find_all('td', { 'class': ['ac seedersInfo', 'ac leechersInfo']})
```

Comment:
I'd probably split this into 2 find calls explicitly for seeders, then leechers.

Reply:
I was just doing what was already there, but I can split it.

Reply:
Splitting this cause an error, so I am going to leave this how it was.